src/framework/components/rigid-body/system.js

```diff
@@ -163,7 +181,33 @@ class SingleContactResult {
 }
 
 /**
- * Object holding the result of a contact between two Entities.
+ * Represents a single point of contact between two colliding rigid bodies in the physics
+ * simulation. Each contact point stores detailed spatial information about the collision,
+ * including both local and world space coordinates of the exact contact points on both entities,
+ * the contact normal  direction, and the collision impulse force.
+ *
+ * Contact points are generated by the physics engine during collision detection and are typically
+ * accessed through a {@link ContactResult} object, which can contain multiple contact points for a
+ * single collision between two entities. Multiple contact points commonly occur when objects
+ * collide along edges or faces rather than at a single point.
```

Comment:
Can't recall Ammo in specific, but generally a contact generated per each vertex, intersecting an object. You can ignore this comment, if Ammo averages a point.

Reply:
I'm not quite sure what you're proposing here. Can you elaborate?

Reply:
Your comment is fine. Thought we could clarify why multiple contacts are generated per single shape. Not necessary.

Reply:
It already says "Multiple contact points commonly occur when objects collide along edges or faces rather than at a single point." Doesn't that clarify things?